Anlab.Mvc/Extensions/EmailSenderExtensions.cs

```diff
@@ -0,0 +1,18 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.Encodings.Web;
+using System.Threading.Tasks;
+using AnlabMvc.Services;
+
+namespace AnlabMvc.Extensions
+{
+    public static class EmailSenderExtensions
+    {
+        public static Task SendEmailConfirmationAsync(this IEmailSender emailSender, string email, string link)
+        {
+            return emailSender.SendEmailAsync(email, "Confirm your email",
+                $"Please confirm your account by clicking this link: <a href='{HtmlEncoder.Default.Encode(link)}'>link</a>");
```

Comment:
@srkirkland We are not using this because we only allow google and ucdavis. Correct? This would be is they were going to create a username/password?

Reply:
@jpknoll correct -- We can re-enable in the case we ever decide to handle username/pass, but for now there will be no way for them to use this since we only support those 2 you mentioned